tests/unit/test_units.py

```diff
@@ -0,0 +1,113 @@
+import numpy as np
```

Comment:
@swillner I generally try and put my imports in 3 levels, each separated by two blank lines:

- packages included with Python
- external packages
- modules from within the package

e.g.

```python
import os


import pytest
import numpy as np


from opensc.units import unit_registry
```

I don't actually mind but might be good to choose a convention and stick to it?

Reply:
ah, ok. compromise: 1 line in between? ;)

Reply:
lol ok